satpy/readers/mersi_l1b.py

```diff
@@ -107,12 +109,12 @@ def get_dataset(self, dataset_id, ds_info):
             data = data * slope + intercept
 
         if dataset_id.get('calibration') == "reflectance":
+            coeffs = self._get_coefficients(ds_info['calibration_key'],
+                                            ds_info['calibration_index'])
+            data = coeffs[0] + coeffs[1] * data + coeffs[2] * data ** 2
             # some bands have 0 counts for the first N columns and
             # seem to be invalid data points
             data = data.where(data != 0)
```

Comment:
This line of code used to be only in the BT calibration function...right? Now it is in the reflectance section?

Reply:
Geez, yes it did. Fixed.